src/pages/partners/affiliate-ib/_faq.tsx

```diff
@@ -35,40 +41,50 @@ const RelativeContainer = styled(SectionContainer)`
     }
 `
 
-const TopLeftDot = styled.img`
+const TopLeftDot = styled.img<DotProps>`
     width: 730px;
     position: absolute;
     top: 4px;
     left: 0;
+    right: 0;
+    margin: ${(props) => (props.is_row ? 'auto' : '0')};
```

Comment:
Let's use destructuring and proper logic here and below
 ```suggestion
    margin: ${({is_row}) => (is_row && 'auto')};
```